CONTRIBUTING.MD

```diff
@@ -14,6 +14,12 @@ To install dependencies, simply run
 pnpm install
 ```
 
+Don't forget to run format using
+
+```sh
+pnpm prettier:write
+```
+
```

Comment:
We have prettier hooks setup, do you think this is still required?

Reply:
oh forgot about that. But someone still forget to format. 🤔

Reply:
:thonk: it's ok to leave it here then

Reply:
We still need that sometimes, I think?